lib/cluster/layer.rb

```diff
@@ -49,13 +49,12 @@ def create
 
     def default_security_group_ids
       [
-        security_group_id_for("#{vpc_name}-InstancesAllowedToAccessEFS"),
-        security_group_id_for("AWS-OpsWorks-Custom-Server")
+        security_group_id_for("#{vpc_name}-OpsworksLayerSecurityGroupCommon"),
```

Comment:
All layers share this "common" security group that opens up ports 0-65535 to the cluster's private IPs + our VPN ips